vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.location.live.map
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.lifecycleScope
+import com.airbnb.mvrx.args
+import com.mapbox.mapboxsdk.maps.MapboxMapOptions
+import com.mapbox.mapboxsdk.maps.SupportMapFragment
+import im.vector.app.R
+import im.vector.app.core.extensions.addFragment
+import im.vector.app.core.platform.VectorBaseFragment
+import im.vector.app.databinding.FragmentLiveLocationMapBinding
+import im.vector.app.features.location.UrlMapProvider
+import javax.inject.Inject
+
+/**
+ * Screen showing a map with all the current users sharing their live location in room.
+ */
+class LocationLiveMapViewFragment @Inject constructor(
```

Comment:
Maybe start using @AndroidEntryPoint for fragment with field injection?

Reply:
Yes, I will try to make the change.

Reply:
See [c46aaa2](https://github.com/vector-im/element-android/pull/6092/commits/c46aaa24a56b0ea531702f3216e3925ce0394ff0)